app/controllers/api/projects_controller.rb

```diff
@@ -84,9 +89,10 @@ def select2ize(data)
       @project_types ||= begin
         types = []
 
+        project_type_to_id = project_source::PERFORMANCE_REPORTING.merge(project_source::RESIDENTIAL_PROJECT_TYPES)
```

Comment:
this was me just trying to make a more complete map of code to ID, since the residential one was missing `:services_only`  (I think) @eanders should I make some change in `Project` here too?